rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/interpreter/program.kt

```diff
@@ -59,7 +59,7 @@ class RpgProgram(val cu: CompilationUnit, dbInterface: DBInterface, val name: St
                 "param ${pv.key} was expected to have type $expectedType. It has value: $coercedValue"
             }
         }
-        interpreter.execute(this.cu, params)
+        interpreter.execute(this.cu, params, reinitialization = true)
```

Comment:
Why force `reinitialization = true`?
Probably, because you have changed also ` Internalnterpreter.execute ` implementation.
Base implementation:
```
fun execute(
        compilationUnit: CompilationUnit,
        initialValues: Map<String, Value>,
        reinitialization: Boolean = true
 )
```
Current implementation:
```
fun execute(
        compilationUnit: CompilationUnit,
        initialValues: Map<String, Value>,
        reinitialization: Boolean
 )
```
This It means to change method behaviour and signature.
Before, the "initialization" argument wasn't mandatory, now it has become mandatory.

It is no clear to me your intent.

Reply:
It's an error... I'll fix it

Reply:
Done, solved by https://github.com/smeup/jariko/pull/165/commits/000252be92866db6d80440bff29ee33bad91c0e2